src/Resources/ResourceFactory.php

```diff
@@ -195,6 +196,54 @@ protected function serviceFactory(array &$properties)
             unset($properties['attributes']['transit_time']['max']);
         }
 
+        if (isset($properties['attributes']['regions_from'][0])) {
+            $regionFromAttribute = $properties['attributes']['regions_from'][0];
+
+            $countryCode = $regionFromAttribute['country_code'];
+            $regionCode = $regionFromAttribute['region_code'];
```

Comment:
Region code might not always be set. If it isn't we should query the API with `region_code` `null`. 
```suggestion
            $regionCode = $regionFromAttribute['region_code'] ?? null;
```